docs/src/modules/java/pages/eventsourced.adoc

```diff
@@ -134,6 +134,53 @@ By default, the generated constructor has an `EventSourcedEntityContext` paramet
 
 When more components are added the `AkkaServerlessFactory` is regenerated and you have to adjust the registration from the `Main` class.
 
+=== Unit Tests with autogenerated Testkit
+
+We can test this Event Sourced Entity implementation with the SchoppingCartTestkit. This class is autogenerated by AkkaServerless when this project is compiled.
+
+The following is a simplification of its contents but in essence, what we show in the next snippet is the signature of the available methods
+
+.target/generated-test-sources/com/example/shoppingcart/domain/ShoppingCartTestKit.java
+[source,java]
+----
+include::example$testkit/java-eventsourced-shopping-cart/intarget/generated-test-sources/akkaserverless/com/example/shoppingcart/domain/ShoppingCartTestKit.java[] 
+----
```

Comment:
I'm not sure it's much value in showing the generated code. That is implementation details that the user shouldn't have to understand. We should focus on how to use it in the docs.

Reply:
ah, it's not the generated code but something representing it. ok

Reply:
I'm still not entirely convinced, better show the generated unit test starting point and only mention the testkit imo.